packages/notifi-react-card/lib/hooks/SubscriptionCardConfig.ts

```diff
@@ -24,6 +24,25 @@ export type BroadcastEventTypeItem = Readonly<{
   tooltipContent?: string;
 }>;
 
+export type HealthCheckEventTypeItem = Readonly<{
```

Comment:
Nothing to change: We need a type library that we can use between the config tool and SDK. No need to build it out now but something that would really help if we ever have time to build it out in the future (key word is 'ever' which pretty means 'never', haha).